src/utility/time/TimeService.cpp

```diff
@@ -26,6 +26,10 @@
 #include "NTPUtils.h"
 #include "AIoTC_Const.h"
 
+#ifdef USE_NOTECARD
```

Comment:
```suggestion
#if defined(USE_NOTECARD) || defined(ARDUINO_ARCH_ESP8266)
```

Reply:
Since they are based on architecture, should they all be `#elif` instead of independent `#if` checks?